service/orchestrator/toe.go

```diff
@@ -408,7 +409,19 @@ func getControls(controls []*orchestrator.Control, levels []string, level string
 		c = append(c, high...)
 	}
 
-	return c, nil
+	// Add parent controls to the sub-control included in the list. That results in duplicates that we can remove later.
```

Comment:
Stupid question: Do the "parent" controls have the "assurance level" set? Because if its empty, you could just add all the controls that do not have an assurance level and you would  only need to do it once and not loop the controls again.

Reply:
This would basically add the semantic that a control with an empty assurance level will be added regardless of the assurance level chosen in the ToE

Reply:
Maybe I don't understand it correctly, but then I have also to loop again and find the controls which do not have sub-controls in the list.